scripts/core/INTRO.rst

```diff
@@ -207,23 +207,23 @@ By default, there is a guarantee that *error* messages are flushed immediately.
 Loggers redirect messages to *stdout*, *stderr*, or a file (default: *stderr*).
 
 All of these logging options can be set with **UR_LOG_LOADER** and **UR_LOG_NULL** environment variables described in the **Environment Variables** section below.
-Both of these environment variables have the same syntax for setting logger options::
-
-  "[level:debug|info|warning|error];[flush:<debug|info|warning|error>];[output:stdout|stderr|file,<path>]"
-  
-  * level - a log level, meaning that only messages from this level and above are printed,
-            possible values, from the lowest level to the highest one: *debug*, *info*, *warning*, *error*,
-  * flush - a flush level, meaning that messages at this level and above are guaranteed to be flushed immediately,
-            possible values are the same as above,
-  * output - indicates where messages should be printed,
-             possible values are: *stdout*, *stderr* and *file*,
-             when providing a *file* output option, a *<path>* is required
-
-  .. note::
-    For output to file, a path to the file have to be provided after a comma, like in the example above. The path has to exist, file will be created if not existing. 
-    All these three logger options are optional. The defaults are set when options are not provided in the environment variable.
-    Options have to be separated with `;`, option names, and their values with `:`. Additionally, when providing *file* output, the keyword *file* and a path to a file
-    have to be separated by `,`.
+Both of these environment variables have the same syntax for setting logger options:
+
+  | "[level:debug|info|warning|error];[flush:<debug|info|warning|error>];[output:stdout|stderr|file,<path>]"
+  |
+  | * level - a log level, meaning that only messages from this level and above are printed,
```

Comment:
These `*` are not being rendered as a bullet list with these `|` line prefixes.

Reply:
Reverted these changes.